Bio/Crystal/__init__.py

```diff
@@ -61,19 +87,23 @@ def __init__(self, data):
         self.data = data[:].lower()
 
     def __eq__(self, other):
+        """No required."""
```

Comment:
What does this mean, and why have you added it to all the methods?

Reply:
That relates to #2116 , so we do not have people updating the Docstrings of magic methods when is going to be deprecated.

Reply:
Yes, but personally I would say no docstring is better than this unhelpful string applied to all of them. Please either take it out again, or add meaningful docstrings if you prefer.